ui/raidboss/data/06-ew/ultimate/dragonsongs_reprise_ultimate.txt

```diff
@@ -179,12 +188,241 @@ hideall "--sync--"
+3050.3 "Empty Dimension" sync / 1[56]:[^:]*:Ser Grinnaux:62DA:/
+3051.7 "Altar Flare 4" #sync / 1[56]:[^:]*:Ser Charibert:63E5:/
+3057.4 "Ancient Quaga" sync / 1[56]:[^:]*:King Thordan:63C6:/
+3067.8 "Heavenly Heel" sync / 1[56]:[^:]*:King Thordan:63C7:/
+3071.0 "Ascalon's Might 1" #sync / 1[56]:[^:]*:King Thordan:63C5:/
+3072.6 "Ascalon's Might 2" #sync / 1[56]:[^:]*:King Thordan:63C5:/
+3074.2 "Ascalon's Might 3" #sync / 1[56]:[^:]*:King Thordan:63C5:/
+3077.0 "--sync--" sync / 1[56]:[^:]*:King Thordan:63C4:/
+3078.1 "Knights of the Round" sync / 1[56]:[^:]*:King Thordan:63ED:/
+3085.2 "Death of the Heavens" sync / 1[56]:[^:]*:King Thordan:6B92:/
+3092.2 "Deathstorm" sync / 1[56]:[^:]*:Darkscale:6B94:/
+3100.4 "Heavy Impact 1" sync / 1[56]:[^:]*:Ser Guerrique:63D6:/
+3102.0 "Lightning Storm" sync / 1[56]:[^:]*:King Thordan:63CC:/
+3102.3 "Twisting Dive" #sync / 1[56]:[^:]*:Vedrfolnir:6B8B:/
+3102.3 "Cauterize" #sync / 1[56]:[^:]*:Darkscale:6B8D:/
+3102.3 "Spear of the Fury" #sync / 1[56]:[^:]*:Ser Zephirin:6B93:/
+3102.3 "Heavy Impact 2" #sync / 1[56]:[^:]*:Ser Guerrique:63D7:/
+3102.4 "Lightning Storm" #sync / 1[56]:[^:]*:King Thordan:63CD:/
+3104.1 "Heavy Impact 3" sync / 1[56]:[^:]*:Ser Guerrique:63D8:/
+3106.0 "Heavy Impact 4" sync / 1[56]:[^:]*:Ser Guerrique:63D9:/
+3107.9 "Heavy Impact 5" sync / 1[56]:[^:]*:Ser Guerrique:63DA:/
+3114.4 "The Dragon's Gaze" sync / 1[56]:[^:]*:King Thordan:63D0:/
+3115.5 "Faith Unmoving" #sync / 1[56]:[^:]*:Ser Grinnaux:62DC:/
+3115.5 "The Dragon's Gaze" #sync / 1[56]:[^:]*:King Thordan:63D1:/
+3115.5 "The Dragon's Glory" #sync / 1[56]:[^:]*:King Thordan:63D2:/
+3117.4 "Heavensflame" sync / 1[56]:[^:]*:Ser Charibert:62DE:/
+3140.3 "Ancient Quaga" sync / 1[56]:[^:]*:King Thordan:63C6:/
+3150.5 "Heavenly Heel" sync / 1[56]:[^:]*:King Thordan:63C7:/
+3153.6 "Ascalon's Might 1" #sync / 1[56]:[^:]*:King Thordan:63C5:/
+3155.2 "Ascalon's Might 2" #sync / 1[56]:[^:]*:King Thordan:63C5:/
+3156.8 "Ascalon's Might 3" #sync / 1[56]:[^:]*:King Thordan:63C5:/
+# Enrage?
+
+
+# Phase 6: Nidhogg v2
+3500.0 "--sync--" sync / 1[56]:[^:]*:King Thordan:6667:/ window 500,0
+3535.0 "Great Wyrmsbreath" sync / 1[56]:[^:]*:Hraesvelgr:6D35:/
+3535.0 "Dread Wyrmsbreath" #sync / 1[56]:[^:]*:Nidhogg:6D33:/
+3535.7 "Swirling Blizzard" #sync / 1[56]:[^:]*:Hraesvelgr:6D38:/
+3535.8 "Ice Breath" #sync / 1[56]:[^:]*:Hraesvelgr:6D37:/
+3535.8 "Holy Breath" #sync / 1[56]:[^:]*:Hraesvelgr:6D3C:/
+3535.8 "Flame Breath" #sync / 1[56]:[^:]*:Nidhogg:6D36:/
+3535.8 "Staggering Breath" #sync / 1[56]:[^:]*:Nidhogg:6D3D:/
+3543.0 "Mortal Vow" sync / 1[56]:[^:]*:Nidhogg:6D30:/
+3554.3 "Akh Afah" sync / 1[56]:[^:]*:Hraesvelgr:6D42:/
+3567.4 "Hallowed Wings" sync / 1[56]:[^:]*:Hraesvelgr:6D24:/
+3568.5 "Cauterize" sync / 1[56]:[^:]*:Nidhogg:6D3E:/
+3567.8 "Hallowed Wings" sync / 1[56]:[^:]*:Hraesvelgr:6D27:/
+3568.6 "Hallowed Plume" #sync / 1[56]:[^:]*:Hraesvelgr:6D29:/
+3568.6 "Hallowed Wings" #sync / 1[56]:[^:]*:Hraesvelgr:6D28:/
```

Comment:
6D29 is the Wide AoE tankbusters baited on the 2 closest/furthest players depending on Hraesvelgr’s head during Hallowed Wings.
Head Down = Closest targets are hit.
Head Up = Furthest targets are hit.

I'm not sure how to tell from the log if the head is up or down.

6D27 is a cast onto self, and 6D28 follows up after. I think. I'll have to look at more logs for this spell, but I would guess 6D27 is left/right and then 6D26 may be the opposite of 6D27. The 6D28 and 6D29 are probably when the damage is applied (if someone were to be standing in the wing's path).

Likewise with 6D29, it could be that there is another spell I am missing and the log I looked at had both head down / head up.

Reply:
Hallowed Wings is more complicated... the first spell indicates head and what side of wing. The second spell is the wing itself casting.

6D23 => 6D25 Head Down, Left Wing
6D24 => 6D25 Head Up, Left Wing
6D26 => 6D28 Head Down, Right Wing
6D27 => 6D28 Head Up, Right Wing